entries/d/datto.com.json

```diff
@@ -8,11 +8,24 @@
     "custom-software": [
       "Duo"
     ],
-    "documentation": "https://help.datto.com/s/article/KB360032979352",
+    "documentation": "https://businessmanagement.datto.com/help/Content/kb/partner-portal/KB360032979352.html",
     "categories": [
       "remote",
       "iot",
       "backup"
+    ],
+    "regions": [
+      "au",
+      "ca",
+      "de",
+      "dk",
+      "fr",
+      "ie",
+      "it",
+      "nl",
+      "se",
+      "uk",
```

Comment:
The UK's ISO 3166-2 country code is `gb`.

Reply:
done ✅